### ISSUE TITLE
Revise structure of metadata chapter

### ISSUE THREAD

**Issue by kepper** [music-encoding/music-encoding#1413]
@axgeertinger, @irmlindcapelle and I where thinking to adjust the structure of the metadata chapter to the following: 

- Intro
- Structure of the MEI Header (here we explain the different areas of the header)
- Works
- Sources
- The MEI file (`<fileDesc>`, `<revisionDesc>`)
- FRBR
- Authority Files
- Common Concepts (here we explain things like `<respStmt>`
- Typical Use Cases (here we plan to give best practice recommendations etc.)

The idea is to take a more user-oriented approach, and everytime we run into models that are used in various places, we only explain the specific use in this very position, but for the model itself, we just reference the "Common Concepts" chapter. 

This one relates to music-encoding/guidelines#82. If we don't get negative feedback, we'll start to implement this in the next couple of weeks.

**Comment by musicEnfanthen** [comment]
@kepper Is the proposed change still intended? Or deprecated by later changes/discussions?

**Comment by musicEnfanthen** [comment]
@kepper @riedde @doerners I guess this can be closed as deprecated?

**Comment by riedde** [comment]
@musicEnfanthen not really. The current tasks improve the "Sources" Chapter but do not revise the structure of the Metadata Chapter (3) itself.

I like the idea of a more user-oriented structure. Should be discussed at the next meeting of the IG

**Comment by doerners** [comment]
I'm with @riedde on this one and would propose to leave it open - at least for now

**Comment by musicEnfanthen** [comment]
Ok, thanks for checking and clarification. Looking forward to the outcome of your IG discussions.

**Comment by musicEnfanthen** [comment]
Transferring to main repo.

**Comment by riedde** [comment]
The @music-encoding/ig-metadata will think about this in the upcoming meeting :)

**Comment by riedde** [comment]
The result from today's meeting of @music-encoding/ig-metadata is to close this issue. Reorganizing the chapter is fine, but we think that this will force a rewirting of the whole chapter. Sadly we do not have the capacity to do this. Maybe this is a task for @music-encoding/ig-digital-pedagogy ?

**Comment by jessgrimmer** [comment]
Dear Dennis,

Thanks so much for being in touch with this. We've discussed, and while
we're happy to provide commentary from a pedagogy perspective, we wouldn't
feel comfortable rewriting the chapter that would affect so many metadata
librarians and their day to day work. Because we're not metadata
specialists, it would be unsuitable for us to take care of this very
critical task.

We're presently engaged in workshop design for several upcoming conferences
as well as finding partners and grant opportunities for the K-12 space. Our
efforts have been decidedly more outward facing as we continue to provide
inroads for music scholars and educators for all levels (not just
university).

All the best,
Joshua & Jessica
